### PR TITLE
Move service-accounts to be namespaced and use modules

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by thomasmckay** [comment]
ack

**Comment by thomasmckay** [comment]
un-ack ... httpd not spinning up

**Comment by ehelms** [comment]
@thomasmckay Made some updates if you want to re-test

**Comment by ehelms** [comment]
[test containers]

**Comment by thomasmckay** [comment]
@ehelms - will do tonight, thanks for the ping

File: containers/deploy/service-accounts/tasks/main.yml [hunk 1]
```diff
@@ -1,39 +1,85 @@
...
+      roleRef:
+        kind: ClusterRole
+        name: "{{ project_name }}-secret-editor"
+      subjects:
+        - kind: ServiceAccount
+          name: "{{ project_name }}-anyuid"
+          namespace: "{{ project_name }}"
+
+- name: "Login as admin"
+  command: oc login -u developer -p a
```
Comment (thomasmckay) [inline containers/deploy/service-accounts/tasks/main.yml]: This should be _system:admin_ login

**Comment by thomasmckay** [comment]
ack pending admin login fix. all pods came up and ui accessible

**Comment by ehelms** [comment]
@thomasmckay Checkout https://github.com/theforeman/forklift/pull/774 to address both sets of concerns

**Comment by thomasmckay** [comment]
ack - all pods happy!

**Pull request merged**